src/geom/mod.rs

```diff
@@ -52,3 +54,21 @@ fn neighbors(cell: crate::CellIndex, scratchpad: &mut [u64]) -> usize {
 
     count
 }
+
+/// Return the geometry of this cell, if it crosses the trans-meridian two polygons are returned.
+///
+/// # Example
+///
+/// ```
+/// let cell = h3o::CellIndex::try_from(0x8a1fb46622dffff)?;
+/// let geom = geo::MultiPolygon::from(cell);
+/// # Ok::<(), h3o::error::InvalidCellIndex>(())
+/// ```
+impl From<crate::CellIndex> for geo::MultiPolygon {
+    fn from(cell: crate::CellIndex) -> Self {
+        let mut polygons = cell_boundary(cell);
+        // converts back everything to degrees
+        polygons.to_degrees_in_place();
+        polygons
+    }
+}
```

Comment:
One last nitpick: can you keep it in `src/index/cell.rs` like the original `From` was?